spring-petclinic/src/main/java/org/springframework/samples/petclinic/service/LoggingAdvice.java

```diff
@@ -0,0 +1,39 @@
+package org.springframework.samples.petclinic.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class LoggingAdvice {
+
+	Logger log = LoggerFactory.getLogger(LoggingAdvice.class);
+
+	@Pointcut(value="execution(* org.springframework.samples.petclinic.*.*.*(..))")
```

Comment:
We can update the pointcut to track the specified business modules:

```suggestion
    @Pointcut("execution(* org.springframework.samples.petclinic.owner.*.*(..)) || execution(* org.springframework.samples.petclinic.vet.*.*(..))")
```